RobotBaseCodes/RobotBaseCodes.ino

```diff
@@ -358,17 +358,29 @@ void IR_reading(IR_SENSOR sensor)
   switch (sensor)
   {
     case LEFT_MID: 
-      SerialCom->println("Mid Left IR Sensor");
-      SerialCom->println(MID_RANGE_LEFT_PIN);
+      SerialCom->print("Mid Left IR Sensor: ");
+      SerialCom->print((pow(2112.5, 1/0.972)) / (pow(analogRead(MID_RANGE_LEFT_PIN),1/0.972))); //MID_RANGE_LEFT_PIN
```

Comment:
can we change expressions like pow(2112.5, 1/0.972) to single constant values? Perhaps you can put some global const variables in at the top then refer to them down here. 

Idk about pow, but division is usually considered pretty computationally expensive and so we want to have the arduino do as little mathematical operations as possible each time it loops.